Changelog.md

```diff
@@ -15,6 +15,7 @@ Bugfixes:
  * SMTChecker: Fix internal error on array slices.
  * SMTChecker: Fix internal error on calling public getter on a state variable of type array (possibly nested) of structs.
  * SMTChecker: Fix internal error on pushing to ``string`` casted to ``bytes``.
+ * AST Import: For constructors, a public visibility is ignored during importing.
```

Comment:
Usually I think I would have missed the order of the entries, but an `A` at the very end is too visible :-D...

Reply:
I'm never sure when we sort the entries. As we go, or before the release?

Reply:
Just always put the entries where they belong. It happens that we mess it up and most of the time have to fix them before the release, but it's always annoying. It's very convenient to see the respective commit on `git blame`ing the Changelog entry and we loose that with, if we just reorder later, so if possible we should avoid it.